deploy/templates/nstemplatetiers/appstudio/spacerole_viewer.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: appstudio-spacerole-viewer # name is used in e2e tests
+objects:
+
+# RoleBinding that grants limited CRUD permissions on AppStudio components CRDs & secrets to the user's SA
+# Role(s) and RoleBinding(s) that grant limited CRUD permissions on AppStudio components CRDs & secrets to the user's SA
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    namespace: ${NAMESPACE}
+    name: appstudio-viewer-user-actions
+  rules:
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - applications
+    - components
+    - componentdetectionqueries
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - promotionruns
+    - snapshotenvironmentbindings
+    - snapshots
+    - environments
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - deploymenttargets
+    - deploymenttargetclaims
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - managed-gitops.redhat.com
+    resources:
+    - gitopsdeployments
+    - gitopsdeploymentmanagedenvironments
+    - gitopsdeploymentrepositorycredentials
+    - gitopsdeploymentsyncruns
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - tekton.dev
+    resources:
+    - pipelineruns
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - results.tekton.dev
+    resources:
+    - results
+    - records
+    - logs
+    verbs:
+    - get
+    - list
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - integrationtestscenarios
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - enterprisecontractpolicies
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - releases
+    - releasestrategies
+    - releaseplans
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - releaseplanadmissions
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - jvmbuildservice.io
+    resources:
+    - jbsconfigs
+    - artifactbuilds
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - spiaccesstokenbindings
+    - spiaccesschecks
+    - spiaccesstokens
+    - spifilecontentrequests
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - appstudio.redhat.com
+    resources:
+    - remotesecrets
+    verbs:
+    - get
+    - list
+    - watch
+  - apiGroups:
+    - ''
+    resources:
+    - configmaps
+    verbs:
+    - get
+    - list
+    - watch
```

Comment:
IIRC, this should be granted automatically via the aggregated `view` ClusterRole

Reply:
should we remove it from here and from https://github.com/codeready-toolchain/host-operator/blob/ea6e92e187b044a11527bdbb33e369f086befa3d/deploy/templates/nstemplatetiers/appstudio/spacerole_contributor.yaml#L138 ? If that's the case, I'd say we can merge this PR as it is and remove from both the roles in a subsequent PR. WDYT?

Reply:
yup, sure. 
as I mentioned in the description of the review - that was just FYI